tutorials/dneb/O.psf

```diff
@@ -0,0 +1,1821 @@
+ O  ca nrl nc  
```

Comment:
I don't think we should mix these things here... This is better suited for Siesta.